packages/suite/src/views/wallet/staking/components/StakingDashboard/StakingDashboard.tsx

```diff
@@ -5,21 +5,25 @@ import { WalletLayout } from 'src/components/wallet';
 import { useSelector } from 'src/hooks/suite';
 
 import { EmptyStakingCard } from './components/EmptyStakingCard';
-import { StakingDashboard } from './components/StakingDashboard';
+import { StakingDashboard as EthStakingDashboard } from '../EthStakingDashboard/components/StakingDashboard';
```

Comment:
could you please call the file and component EthStakingDashboard?

Reply:
https://github.com/trezor/trezor-suite/pull/15409/commits/f3c63c5420d93693d93d4758b4ced0b84df35aad